### PR TITLE
[plsql] updates should allow for multiple statements

### PR DESCRIPTION
**Pull request by tashiscool**
Before submitting a PR, please check that:
 - [x] The PR is submitted against `master`. The PMD team will merge back to support branches as needed.
 - [x] `./mvnw clean verify` passes. This will [build](https://github.com/pmd/pmd/blob/master/BUILDING.md) and test PMD, execute PMD and checkstyle rules. [Check this for more info](https://github.com/pmd/pmd/blob/master/CONTRIBUTING.md#code-style)

**PR Description:**

### CODE REVIEW

**Comment by jsotuyod** [comment]
@oowekyala you can add `?w=1` to the url to ignore whitespace in the diff, ie: https://github.com/pmd/pmd/pull/1554/files?w=1

**Comment by oowekyala** [comment]
@jsotuyod so cool, thanks!

**Comment by pmd-test** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR cannot be merged yet.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_danger">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: pmd-plsql/etc/grammar/PldocAST.jjt [hunk 1]
```diff
@@ -1904,8 +1904,10 @@ ASTUpdateSetClause UpdateSetClause() :
     <SET>
     (
         <ROW> "=" ID()
+        ( "," Column() "=" ( Expression() | <_DEFAULT> )*)*
```
Comment (adangel) [inline pmd-plsql/etc/grammar/PldocAST.jjt]: Is this correct? I didn't find a clear description, whether a ROW can be combined with single columns at the same time. The ROW uses a record, which contains all columns anyway...

So: Is this actually valid syntax?

```
UPDATE schedule
  SET ROW = default_week,
    Tue = 'Day Off'
  WHERE week = 1;
```

**Comment by adangel** [comment]
Thanks for your contribution! This is merged and will be part of PMD 6.11.0.

**Pull request merged**